src/components/panels/ConfigPanel/_config-panel.scss

```diff
@@ -129,6 +142,10 @@
     @media screen and (max-width: $breakpoint-x-small) {
       padding-right: 0;
     }
+
+    button {
```

Comment:
Can you target a class here in case someone updates the mark up.

Reply:
Done!